bin/NativeTests/JsRTApiTest.cpp

```diff
@@ -1243,7 +1244,8 @@ namespace JsRTApiTest
             size_t length;
             REQUIRE(JsStringToPointer(nameValue, &name, &length) == JsNoError);
 
-            CHECK(length == 1);
+            REQUIRE(length == 1);
+#pragma prefast(suppress:__WARNING_MAYBE_UNINIT_VAR, "The require on the previous line should ensure that name[0] is initialized")
```

Comment:
>__WARNING_MAYBE_UNINIT_VAR [](start = 25, length = 26)

Did not all warnings have these fancy named macros? (There are some you're still suppressing or disabling by number)

Reply:
Nope, not all of them do. There are a couple cases where I missed using the named macro, though, that I should be fixing up in the next set of commits.